src/views/Help/Section508/TestingTemplate/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import HelpBreadcrumb from 'components/HelpBreadcrumb';
+import MainContent from 'components/MainContent';
+import RelatedArticles from 'components/RelatedArticles';
+import TestingTemplatesBase from 'components/TestingTemplates';
+
+const TestingTemplates = () => {
+  return (
+    <>
+      <MainContent className="grid-container">
+        <HelpBreadcrumb type="Close tab" />
+        <TestingTemplatesBase helpArticle />
+      </MainContent>
+      <div className="margin-top-7">
```

Comment:
Looks like the spacing between this div and `<MainContent>` is about double what it needs to be because of the bottom margin on the last `<SummaryBox>` in `<TestingTemplatesBase>`.

Also this is just a nitpick, but you could get rid of the nested divs by adding your utility class to the `<MainContent>` component instead of wrapping `<RelatedArticles>` in an extra div.